src/components/Form/Internal/useForm.ts

```diff
@@ -80,18 +80,18 @@ export class FormStore {
 
     public getForm = (): InternalOcFormInstance => ({
         getFieldValue: this.getFieldValue,
-        getFieldsValue: this.getFieldsValue,
+        getFieldListValues: this.getFieldListValues,
         getFieldError: this.getFieldError,
         getFieldWarning: this.getFieldWarning,
-        getFieldsError: this.getFieldsError,
-        isFieldsTouched: this.isFieldsTouched,
-        isFieldTouched: this.isFieldTouched,
-        isFieldValidating: this.isFieldValidating,
-        isFieldsValidating: this.isFieldsValidating,
+        getFieldListErrors: this.getFieldListErrors,
+        isListTouched: this.isListTouched,
+        isTouched: this.isTouched,
+        isValidating: this.isValidating,
```

Comment:
These are "Field" specific? or "Form"?

Reply:
Form, the diff is how they are called. form.isTouched('name') (public) vs field.isTouched() (private used by the useForm method pointing to that field)